packer/resources/features/logstash/install.sh

```diff
@@ -18,9 +18,22 @@ ln -s /opt/features/logstash/logstash /opt/logstash
 # TODO could use the handy substitution script once #23 is merged
 ESCAPED_USER=$(echo $USER | sed -e 's/[\/&]/\\&/g')
 ESCAPED_CONFIGFILE=$(echo $CONFIGFILE | sed -e 's/[\/&]/\\&/g')
+
+# Upstart
+if [ -d "/etc/init" ]; then
 sed \
   -e "s/@USER@/$ESCAPED_USER/g" \
   -e "s/@CONFIGFILE@/$ESCAPED_CONFIGFILE/g" \
   /opt/features/logstash/logstash.conf.template > /etc/init/logstash.conf
+  echo "Logstash is now installed as an Upstart service. You can start it by running 'sudo start logstash'."
+fi
+
+# systemd
+if [ -d "/etc/systemd/system" ]; then
+sed \
+  -e "s/@USER@/$ESCAPED_USER/g" \
+  -e "s/@CONFIGFILE@/$ESCAPED_CONFIGFILE/g" \
+  /opt/features/logstash/logstash.service.template > /etc/systemd/system/logstash.service
```

Comment:
Can @sihil or @philwills confirm that `/etc/systemd/system` is a reasonable place to put the service file?